content/SCALE/SCALEUIReference/Credentials/Certificates/CAScreensSCALE.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Certificates Authorities Screens"
+description: "This article provides information on SCALE certificate authroities screens and settings."
+weight: 20
+aliases:
+tags:
+ - scalecas
+ - scalecsrs
+ - scalecertificates
+---
+
+{{< toc >}}
+
+The **Certificate Authorities** widget on the **Credentials > Certificates** screen displays certificate authorities(CAs) added to SCALE, and allows you to add new CAs, or download, delete, or edit the name of an existing CA. 
+
+![CertificateAuthoritiesWidgetNoCAs](/images/SCALE/22.02/CertificateAuthoritiesWidgetNoCAs.png "Certificate Authorities Widget No CAs")
+
+The <iconify-icon icon="teenyicons:download-solid"></iconify-icon> download icon downloads the CA to your server.
+
+<span class="material-icons">delete</span> deletes the CA from your server.
+
+Each CA listed on the widget is a link that opens the **Edit CA** screen.
+
+**Add** opens the **[Add CA](#add-ca-wizard-screens)** wizard that steps you through setting up a certificate authority (CA) that certifies the ownership of a public key by the named subject of the certificate. 
+
+## Add CA Wizard Screens
+The **Add CA** wizard screens step users through configuring a new certificate authority on TrueNAS SCALE. 
+The wizard has five different configuration screens, one for each step in the CA configuration process:
+
+1 **[Identifier and Type](#identifier-and-type-options)**
+
+2 **[Certificate Options](#certificate-options)**
+
+3 **[Certificate Subject](#certificate-subject-options)**
+
+4 **[Extra Constraints](#extra-constraints-options)**
+
+5 **[Confirm Options](#confirm-options)**
+
+### Identifier and Type Options
+The **Identifier and Type** options specify the CA name and choose whether to create a new CA or import an existing CA.     
+Users can also select a predefined certificate extension from the **Profiles** dropdown list.
+{{< expand "Click Here for More Information" "v" >}}
```

Comment:
```suggestion
{{< expand "Click Here for More Identifier and Type Information" "v" >}}
```

Reply:
No because this is a generic expand title used for all the expands for screens.